lib/types/bcachefs.nix

```diff
@@ -66,7 +66,7 @@
       default = ''
         # Write device arguments to temporary directory for bcachefs_filesystem.
         {
-          printf '%s\n' '--label="${config.label}"';
```

Comment:
Maybe `printf '%q\n' '--label=${config.label}'` to make sure it gets shell quoted?